python/py-commpy/Portfile

```diff
@@ -33,7 +33,8 @@ if {${name} ne ${subport}} {
     depends_lib-append \
         port:py${python.version}-matplotlib \
         port:py${python.version}-numpy \
-        port:py${python.version}-scipy
+        port:py${python.version}-scipy \
+        port:py${python.version}-nose
```

Comment:
nose is only a test dependency so it should not be added under “depends_lib”

Reply:
right; I forgot this point; I need to write a note in the Portfile. thanks!